components/adminTable.js

```diff
@@ -44,12 +47,24 @@ export default function AdminTable(props) {
   const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
     useTable({ columns, data });
 
+  const paginatedRows = React.useMemo(
+    () =>
+      rows.slice(pageIndex * entriesPerPage, (pageIndex + 1) * entriesPerPage),
+    [rows, pageIndex, entriesPerPage]
+  );
+
+  const startEntry = pageIndex * entriesPerPage + 1;
+  const endEntry = Math.min((pageIndex + 1) * entriesPerPage, rows.length);
+  const totalEntries = rows.length;
```

Comment:
We should leverage this variable for the pagination feature, we should be able to dynamically create the options for the footer, removing the hardcoded options. We can just generate some mock data to fill in the students in a class so that we can simulate the footer still displaying and working correctly?  Let me know your thoughts on this.